src/JsonApi/Serializer/ItemNormalizer.php

```diff
@@ -68,7 +68,7 @@ public function normalize($object, $format = null, array $context = [])
         }
 
         $outputClass = $this->getOutputClass($this->getObjectClass($object), $context);
-        if (null !== $outputClass && null !== $this->dataTransformer && $this->dataTransformer->supportsTransformation($object, $context)) {
+        if (null !== $outputClass && null !== $this->dataTransformer && $this->dataTransformer->supportsTransformation($object, $outputClass, $context)) {
             $object = $this->dataTransformer->transform($object, $context);
```

Comment:
https://github.com/api-platform/core/pull/2543 also fixes this line, thanks for the contribution though and sorry I saw that a bit late

Reply:
No problem, glas to help ;)